src/pages/Captures.tsx

```diff
@@ -1,12 +1,35 @@
-import CaptureRoot from 'components/capture/Root';
+import AddIcon from '@mui/icons-material/Add';
+import { Button, Toolbar } from '@mui/material';
+import CatalogList from 'components/CatalogList';
```

Comment:
Would the _src/components/shared_ directory be a better location for the highlighted file?